test/absinthe/execution/subscription_test.exs

```diff
@@ -77,6 +77,10 @@ defmodule Absinthe.Execution.SubscriptionTest do
       # this pubsub is local and doesn't support clusters
       :ok
     end
+
+    def list_registry_keys() do
+      Registry.keys(__MODULE__, self())
```

Comment:
This is the registry name that is used to store the subscription entries internally
```suggestion
      Registry.keys(__MODULE__.Registry, self())
```